docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
+sphinx<3
```

Comment:
I think we need to specify a minimum version, otherwise the current readthedocs one is <3 so it won't be upgraded.
Also I think its better to stick to a single one, such as sphinx==2.0.1.
Thanks!

Reply:
Okay!